src/beignet/nn/functional/_torsion_angle_loss.py

```diff
@@ -0,0 +1,36 @@
+from typing import Tuple
+
+import torch
+from torch import Tensor
+
+
+def torsion_angle_loss(input, target: Tuple[Tensor, Tensor]) -> Tensor:
+    """
+
+    Parameters
+    ----------
+    input
+    target
+
+    Returns
+    -------
+
+    """
+    a = input / torch.norm(input, dim=-1, keepdim=True)
+
+    b, c = target
+
+    x = torch.mean(
```

Comment:
Can we call `x` and `y` something slightly more descriptive like `loss_torsion` and `loss_anglenorm` to make it easier to compare with the description in the supplement?